salt/environment_settings.yml

```diff
@@ -314,7 +314,10 @@ environments:
     purposes:
       odl-video-service:
         num_instances: 3
-        domain: video.odl.mit.edu
+        domain:
```

Comment:
We should change this to be plurarl

Reply:
I had that as plural initially, but changed it back since it means we would need to change all occurrences of it in that file so that when we call it from elsewhere, it would be one or the other. Right?

Reply:
I think that it makes sense to just commit to it always being pluarl